code/src/components/Feed.js

```diff
@@ -0,0 +1,86 @@
+/* eslint no-underscore-dangle: 0 */
+import React, { useEffect, useState } from 'react'
+
+import GeneratedFeed from 'components/GeneratedFeed.js';
+import PostNewThought from 'components/PostNewThought';
+import SiteHeader from 'components/SiteHeader';
+
+const Feed = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [newThought, setNewThought] = useState('')
+  const [counter, setCounter] = useState(0)
+  const [loading, setLoading] = useState(false)
```

Comment:
It's hard to tell if the loading is working since the data loads fast, but I don't see anything that could be wrong. How do you know that it's not working?